PublishedModules/glueckself/mpdclient/mpdclient.install

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "mpdclient",
+	"version": 0.1,
+	"author": "glueckself",
+	"maintainers": ["glueckself"],
+	"desc": "Control an MPD server",
+	"aliceMinVersion": 0.10,
+	"systemRequirements": [],
+	"pipRequirements": ["python-mpd2"],
+	"conditions": {
+		"lang": [
+			"en",
+			"de"
+		],
+		"online": false
```

Comment:
This means people that allow Alice to use internet services won't be able to use your module, I guess that's not intended?

Reply:
No it's not, thank you.
I thought that meant that the module requires online connection (stayCompletlyOffline has to be false). I'll fix that later today.